Running is broken when test has no output

Creating a test case without output and then trying to run your task solution  breaks.

```
❯ egor  r
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/chermehdi/egor/commands.RunAction(0xc000098580, 0x0, 0x0)
	.../go/src/github.com/go-egor/commands/run.go:512 +0x68c
github.com/urfave/cli/v2.(*Command).Run(0x199fa80, 0xc000098380, 0x0, 0x0)
	.../go/pkg/mod/github.com/urfave/cli/v2@v2.1.1/command.go:161 +0x4e0
github.com/urfave/cli/v2.(*App).RunContext(0xc000082300, 0x163e4a0, 0xc000028320, 0xc00000e0a0, 0x2, 0x2, 0x0, 0x0)
	...go/pkg/mod/github.com/urfave/cli/v2@v2.1.1/app.go:302 +0x814
github.com/urfave/cli/v2.(*App).Run(...)
	.../go/pkg/mod/github.com/urfave/cli/v2@v2.1.1/app.go:211
main.main()
...
```

--- a/commands/case.go
+++ b/commands/case.go
@@ -111,15 +111,17 @@ func CustomCaseAction(context *cli.Context) error {
 		return err
 	}
 
+	outputLines := []string{}
 	if !context.Bool("no-output") {
 		color.Green("Provide your output:")
-		outputLines := readFromStdin()
-		metaData, err = AddNewCaseOutput(outputLines, caseName, metaData)
+		outputLines = readFromStdin()
+	}
 
-		if err != nil {
-			color.Red("Failed to add new case output")
-			return err
-		}
+	metaData, err = AddNewCaseOutput(outputLines, caseName, metaData)
+
+	if err != nil {
+		color.Red("Failed to add new case output")
+		return err
 	}
 
 	err = metaData.SaveToFile(path.Join(cwd, configFileName))

